presto-main/src/main/java/com/facebook/presto/execution/resourceGroups/QueryType.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.execution.resourceGroups;
+
+public enum QueryType
+{
+    CREATE_AS,
+    DATA_DEFINITION,
+    DELETE,
+    DESCRIBE,
+    EXPLAIN,
+    INSERT,
```

Comment:
Similarly for CREATE_AS vs INSERT. They could both be seen as a general case of inserting data into a table.